ops/jenkins/bfd-build-deploy/Jenkinsfile

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env groovy
+
+/**
+ * <p>
+ * This script will be run by Jenkins when building deploying apps and app AMIs to ephemeral environments.
```

Comment:
Are there any controls to ensure that it's only ephemeral environments that are subject to this job?

Reply:
No Im curious about this